src/graphql/resolvers/resolver.user.ts

```diff
@@ -0,0 +1,30 @@
+import { PrismaClient } from '@prisma/client';
+import { UserController } from '../../controller/user.controller.js';
+import { UserService } from '../../services/user.service.js';
+import type { CreateUserInput} from '../../types/types.js';
+
+const prisma = new PrismaClient();
+const userService = new UserService(prisma);
+const userController = new UserController(userService);
+
+export const userResolver = {
+  Query: {
+    Users: (): string => {
+      return 'nada';
+    },
+  },
+
+  Mutation: {
+    createUser: async (
+      _: unknown,
+      {
+        data,
+      }: {
+        data: CreateUserInput;
+      },
+    ): Promise<CreateUserInput> => {
```

Comment:
Essa função tá tendo como retorno a mesma tipagem do Input? Será que faz sentido isso?

Reply:
Não faz sentido porque , o mesmo dados de entrada,não são o mesmos dados de saída, tipo o birthDate que entra como string e sai com o formato Date, pelo que estudei do prisma consigo puxar o tipo User que bate com o schema do prisma,e é exatamente esse o retorno que to esperando